functions.php

```diff
@@ -1 +1,15 @@
 <?php
+function showPokemon(array $allPokemon) {
+    $returnedHTML = "";
+    if (count($allPokemon) === count($allPokemon, COUNT_RECURSIVE))  {
+        return 'WRONG INPUT';
+    }
+    foreach($allPokemon as $pokemon) {
+        $returnedHTML .= "<p>" . "Pokedex ID: " . $pokemon['pokedex_id'] . "</p>" .
```

Comment:
no need to concatenate `"<p>" . "Pokedex ID: "` - you can just `"<p>Pokedex ID: "`

Reply:
and same for all the other ones